video/cloud-client/src/test/java/com/example/video/DetectIT.java

```diff
@@ -37,6 +37,7 @@
   static final String LABEL_FILE_LOCATION = "./resources/cat.mp4";
   static final String SHOTS_FILE_LOCATION = "gs://demomaker/gbikes_dinosaur.mp4";
   static final String EXPLICIT_CONTENT_LOCATION =  "gs://demomaker/cat.mp4";
+  static final String SPEECH_GCS_LOCATION = "gs://python-docs-samples-tests/video/googlework_short.mp4";
```

Comment:
nit: is this also in the java-docs-samples GCS bucket?

Reply:
Yes, it is there.